### ISSUE TITLE
Support `vitest.rootPath` option

### ISSUE THREAD

**Issue by mizdra** [vitest-dev/vscode#151]
**Is your feature request related to a problem? Please describe.**
I want to use this extension even in cases where `vitest.config.ts` is not directly under the workspace root directory.

For example, imagine a workspace with the following directory structure.

```
workspace-root/
├ .github/
├ backend/
├ ui/
│ ├ node_modules/
│ ├ src/
│ │ ├ index.ts
│ │ └ index.test.ts
│ ├ package.json
│ └ vitest.config.ts
└ terraform/
```

In this workspace, backend, frontend, and IaC code are clearly separated by directory. vitest is used only for frontend code, so `vitest.config.ts` is under `ui/` directory. Therefore, the extension must run vitest with `ui/` directory as the current directory, not the workspace root directory.

However, this extension does not have an option to change the current directory where vitest runs.

**Describe the solution you'd like**
Add the `vitest.rootPath` option; vscode-jest has a similar option: https://github.com/jest-community/vscode-jest#rootpath

**Describe alternatives you've considered**
Nothing.

**Additional context**
Nothing.

**Comment by eomeara** [comment]
I have same problem and same desired fix.

**Comment by sheremet-va** [comment]
This should be fixed in pre-release 0.5.0 and higher. Note that the extension now requires Vitest 1.4.0 or higher.

The extension now automatically finds the config and uses it as the working directory.
